script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -335,175 +406,196 @@ His screams reverberate throughout the entire room.
 }
 [sha:56dd665bbd801de43e7ab9de30eb48ff8c4ddeb1]{
 -- Page 451, Offset 19390. Glued. Mods: @n.
 -- もちろんそのたびに死に至る激痛が獲物を襲う。内臓をすべて消費した後は耳と鼻をそぎ落とす。変わり果てた自分の姿を見て獲物がモウ殺シテと嘆願する。
-Of course, he's assaulted by a fatal pain every time. When his internal organs have all been gouged out, next comes the shearing of his ears and nose. Seeing his mangled form, he will beg to be killed.
+ Of course, each strike inflicts a lethal dose of pain.
+ Once she's done with his organs, she rips off his nose and ears.
+ If her prey could see what he's become, he'd beg her to just kill him.
 }
 [sha:e3048536b20af97e936beeb8bab957c582d1d408]{
 -- Page 451, Offset 19391. Glued. Mods: @n.
 -- 最後に頭蓋をパックリと割る。大皿のようなその容器に聖水を流し込む。
-Finally, spitting the skull open and pouring holy water into the vessel.
+// Tfw no trepanning nun gf
+ Finally, she punctures a hole in his skull and fills it with holy water.
 }
 [sha:371ff6392a9cccc63a5c349a8d2bdf794e81fe7c]{
 -- Page 451, Offset 19392. Glued. Mods: @n.
 -- 彼等にとって聖水は硫酸に等しい。脳に痛覚はない筈だが、洗礼によって清められた液体は痛覚以上の痛みで獲物を包み込み、
-To them, holy water is like sulfuric acid. The brain shouldn't have a sense of pain, but the liquid purified by baptism envelops the prey with an unimaginable agony and―――
+ To them, holy water is like sulfuric acid.
+ The brain shouldn't have a sense of pain,
+ but this liquid, which has been purified by a baptism,
+ envelops her prey with an unimaginable agony
 }
 [sha:396366a0d3c6f2f1f955e5407404cc87a53bc511]{
 -- Page 451, Offset 19393. Glued.
 -- 発狂させる。
-Drives them insane.
+ and drives them insane.
 }
 [sha:fe0a6cbc12e924faab8b0542d644d47f47bbf17b]{
 -- Page 451, Offset 19394.
 -- コロシテ、コロシテ、コロシテ、と繰り返すだけの蓄音機が完成する。
-// wow this line is stupid
-In the end, he becomes a phonograph, only capable of repeating 'Kill me. kill Me. kill me'.
+The product is a broken record,
+ capable only of repeating the words 'Kill me. kill Me. kill me'.
 }
 [sha:84bd26874c4d10f9174c0438c9bd3ea92daf9f8f]{
 -- Page 452, Offset 19395.
 -- 「――――――、ふう」
-"――――Phew"
+"――――%{i}Phew%{/i}."
 }
 [sha:2d1d92ca6c72ecc27f52847977a85170a9c775bc]{
 -- Page 452, Offset 19396.
 -- はあ、と地獄には場違いな吐息が漏れた。
 // dear lord
-She lets out a sigh so sinister it'd belong in hell.
+She lets out a sigh that'd make the devil himself tremble.
 }
 [sha:39430e8a2d65b078679851de63f2f37e5e0ce639]{
 -- Page 452, Offset 19397. Mods: @n.
 -- シスターは槍を持ち直すと、よいしょ、
-The sister picks up her spear once more and with a 'Heave-ho!'―――
+The nun hefts her spear once more, and with a 'heave-ho!'――
 }
 [sha:00676f0a0ccf9afe07d93d55b1e0d579bad66a76]{
 -- Page 452, Offset 19398. Glued.
 -- とかけ声をこぼして蓄音機を破壊した。
-Destroys the phonogram.
+obliterates the broken record.
 }
 [sha:b4814173d90a006d8f6d539d3313d3fb4bdd49d2]{
 -- Page 453, Offset 19399. Mods: @k, @e.
 -- 「しっぱいしっぱい、あんまりにも面白いコト言われたから趣味に走っちゃった。
-"My bad, my bad. What he was saying was just so funny I got caught up in all the fun.
+// oopsie woopsie!! uwu I made a fucky wucky!! A wittle fucko boingo!
```

Comment:
now now, we don't need to play to the people reading for the comments, do we

Reply:
Only people I'm here to play to are 1. me, losing my mind 4 hours into a file and 2. my darling reviewers